site/docs/v1/tech/guides/breached-password-detection.adoc

```diff
@@ -0,0 +1,274 @@
+---
+layout: doc
+title: Breached Password Detection
+description: Learn how to create a passwordless experience for your end users.
+---
+:page-liquid:
+
+== What Is Breached Password Detection
+
+Breached password detection consists of the following tasks:
+
+. Finding breached and compromised passwords.
+. Building a system to download, process and store these datasets.
+. Checking passwords to see if they've been compromised.
+. Taking action when a leaked credential is found.
+
+FusionAuth has taken care of steps 1, 2 and 3. All you need to do is to configure your breached password detection options.
+
+The end goal, of course, is to have a more secure auth system by avoiding unauthorized access due to any of your users' credentials being compromised.
+
+== How Do I Use Breached Password Detection 
+
+*This is a paid edition feature and is unavailable in the community edition. Please visit link:/pricing[our pricing page] to learn more about paid editions.*
+
+Here's a video showing setup and usage of the breached password detection feature.
+
+video::JUB-5wQ_fdI[youtube,width=560,height=315]
+
+Conceptually, FusionAuth uses a pull model rather than a push model. Passwords are checked only when the user provides, at registration, password change, or, optionally, during login. 
+
+== Setting Up
+
+The first step to enabling breached password detection is to have a valid license key. Please visit link:/pricing[our pricing page] to learn more about paid editions and to buy a license.
+
+The next step is to activate your license key. You'll need to ensure that your FusionAuth instance has outbound network access. Follow the steps outlined in the link:/docs/v1/tech/reactor[Reactor documentation].
+
+Navigate to the [breadcrumb]#Tenants# tab and edit the tenant for which you wish to enable breached password detection. Go to the [breadcrumb]#Password# tab and the [breadcrumb]#Breached password detection settings# section. Click the checkbox to enable breached password detection. This will open up a new set of configuration options, as seen below:
+
+image::guides/breached-password-detection/turn-on-detection.png[Turn on breached password detection,width=700,role=box-shadow margin-right]
+
+The default settings prevent use of leaked passwords in a number of contexts. Simply by enabling breached password detection, all users within this tenant will, from that moment forward, no longer be able to use any password that has been found to be breached by the FusionAuth team.
```

Comment:
Should we use leaked or compromised here? Leaked doesn't seem to fit as well for me, because most of the credentials weren't leaked, they were stolen. At least those are different in my mind.

Reply:
Makes sense.